### PR TITLE
Add Ficus playbooks

### PR DESCRIPTION
**Pull request by mrgnr**
Basic, Pro, and Enterprise tiers.

### CODE REVIEW

**Comment by mrgnr** [comment]
[(3) Ficus: Create playbooks for Basic, Pro, and Enterprise tiers](https://trello.com/c/e8ctij5B/916-3-ficus-create-playbooks-for-basic-pro-and-enterprise-tiers)

File: playbooks/appsemblerPlaybooks/ficus_basic.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      when: "INSTALL_DEMO_DATA"
+    - role: oauth_client_setup
+      when: COMMON_ENABLE_OAUTH_CLIENT
+    - oraclejdk
+    - role: elasticsearch
+      when: "'localhost' in EDXAPP_ELASTIC_SEARCH_CONFIG|map(attribute='host')"
+    - forum
+    - { role: notifier, NOTIFIER_DIGEST_TASK_INTERVAL: "5" }
+    - { role: "xqueue", update_users: True }
+    - certs
```
Comment (melvinsoft) [inline playbooks/appsemblerPlaybooks/ficus_basic.yml]: We're moving away of using PDF certs, and start using HTML certs. For now only Metalogix and ISC are using certs, so maybe we can remove this role or add a conditional in False by default.

File: playbooks/appsemblerPlaybooks/ficus_enterprise.yml [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+# must come after xqueue running
+- name: Install certs on edxapp instances
+  hosts: edxapp-server
+  sudo: True
+  gather_facts: True
+  vars:
+    migrate_db: "yes"
+  roles:
+    - certs
```
Comment (melvinsoft) [inline playbooks/appsemblerPlaybooks/ficus_enterprise.yml]: Same as above, but in this case we can remove the entire play.

File: playbooks/appsemblerPlaybooks/ficus_pro.yml [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+# must come after xqueue running
+- name: Install certs on edxapp instance
+  hosts: edxapp-server
+  sudo: True
+  gather_facts: True
+  vars:
+    migrate_db: "yes"
+  roles:
+    - certs
```
Comment (melvinsoft) [inline playbooks/appsemblerPlaybooks/ficus_pro.yml]: Same as above

File: playbooks/appsemblerPlaybooks/ficus_basic.yml [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    - mysql_init
+    - role: memcache
+      when: "'localhost' in ' '.join(EDXAPP_MEMCACHE)"
+    - role: mongo_3_0
+      when: "'localhost' in EDXAPP_MONGO_HOSTS"
+    - { role: 'rabbitmq', rabbitmq_ip: '127.0.0.1' }
+    - { role: 'edxapp', celery_worker: True }
+    - edxapp
+    - notifier
+    - edx_notes_api
```
Comment (melvinsoft) [inline playbooks/appsemblerPlaybooks/ficus_basic.yml]: Only 1 customer is using notes, and I'm sure if is working ATM. Maybe we can discuss about removing this in our defaults playbooks, or put it behind a conditional.

**Comment by mrgnr** [comment]
@melvinsoft I removed `certs` and made `edx_notes_api` optional.

**Pull request merged**